mne/channels/channels.py

```diff
@@ -403,6 +403,8 @@ def pick_channels(self, ch_names, copy=False):
         -----
         .. versionadded:: 0.9.0
         """
+        if not isinstance(ch_names, (tuple, list, np.ndarray)):
+            ch_names = [ch_names]
```

Comment:
If would just check if is string type and if so raise an error. That's the minimal change 


> On 1 août 2015, at 13:12, Eric Larson <notifications@github.com> wrote:
> 
> In mne/channels/channels.py:
> 
> > @@ -403,6 +403,8 @@ def pick_channels(self, ch_names, copy=False):
> >          -----
> >          .. versionadded:: 0.9.0
> >          """
> > +        if not isinstance(ch_names, (tuple, list, np.ndarray)):
> > +            ch_names = [ch_names]
> I'm fine either way. There are other places that we upconvert a single arg to a list. But if we do it here, we should at least have another check like:
> 
> if not isinstance(ch_names, (tuple, list, np.ndarray)):
>     if not isinstance(ch_names, string_types):
>         raise TypeError(...)
>     ch_names = [ch_names]
> But @agramfort if you don't like the upconversion, we don't need to have it.
> 
> —
> Reply to this email directly or view it on GitHub.
>